File: Flights Application/ClientApp/src/app/api/models/booking-rm.ts
/* tslint:disable */
/* eslint-disable */
import { TimePlaceRm } from '../models/time-place-rm';
export interface BookingRm {
  airline?: string | null;
  arrival?: TimePlaceRm;
  departure?: TimePlaceRm;
  flightId?: string;
  numberofBookedSeats?: number;
  passengerEmail?: string | null;
  price?: string | null;
}

File: Flights Application/ReadModels/BookingRm.cs
namespace Flights_Application.ReadModels
{
    public record BookingRm(
        Guid FlightId,
        string Airline,
        string price,
        TimePlaceRm Arrival,
        TimePlaceRm Departure,
        int NumberofBookedSeats,
        string PassengerEmail
        );
   
}
